Goobi/setup/default.sql

```diff
@@ -83,3 +83,62 @@ INSERT INTO `benutzergruppenmitgliedschaft` (`BenutzerID`,`BenutzerGruppenID`) V
 INSERT INTO `benutzergruppenmitgliedschaft` (`BenutzerID`,`BenutzerGruppenID`) VALUES
  (6,6);
 /*!40000 ALTER TABLE `benutzergruppenmitgliedschaft` ENABLE KEYS */;
+
+/*!40000 ALTER TABLE `metadatenkonfigurationen` DISABLE KEYS */;
+INSERT INTO `metadatenkonfigurationen` (`MetadatenKonfigurationID`, `Titel`, `Datei`, `orderMetadataByRuleset`) VALUES
+(1, 'SLUB', 'slub.xml', b'0');
+/*!40000 ALTER TABLE `metadatenkonfigurationen` ENABLE KEYS */;
+
+/*!40000 ALTER TABLE `dockets` DISABLE KEYS */;
+INSERT INTO `dockets` (`docketID`, `name`, `file`) VALUES
+(1, 'default', 'docket.xsl');
+/*!40000 ALTER TABLE `dockets` ENABLE KEYS */;
+
+/*!40000 ALTER TABLE `projekte` DISABLE KEYS */;
+INSERT INTO `projekte` (`ProjekteID`, `Titel`, `useDmsImport`, `dmsImportTimeOut`, `dmsImportRootPath`, `dmsImportImagesPath`, `dmsImportSuccessPath`, `dmsImportErrorPath`, `dmsImportCreateProcessFolder`, `fileFormatInternal`, `fileFormatDmsExport`, `metsRightsOwner`, `metsRightsOwnerLogo`, `metsRightsOwnerSite`, `metsDigiprovReference`, `metsDigiprovPresentation`, `metsDigiprovReferenceAnchor`, `metsDigiprovPresentationAnchor`, `metsPointerPath`, `metsPointerPathAnchor`, `metsPurl`, `metsContentIDs`, `metsRightsOwnerMail`, `startDate`, `endDate`, `numberOfPages`, `numberOfVolumes`, `projectIsArchived`) VALUES
+(1, 'Example Project', b'1', 3600000, '/usr/local/goobi/hotfolder/', '/usr/local/goobi/hotfolder/', '/usr/local/goobi/success/', '/usr/local/goobi/error_mets/', b'0', 'Mets', 'Mets', 'Digital Library Kitodo', 'http://www.kitodo.org/fileadmin/groups/kitodo/Logo/kitodo_logo_rgb.png', 'http://www.kitodo.org', 'http://opac.sub.uni-goettingen.de/DB=1/PPN?PPN=$(meta.topstruct.CatalogIDDigital)', 'http://www.kitodo.org/resolver?id=$(meta.topstruct.CatalogIDDigital)', 'http://opac.sub.uni-goettingen.de/DB=1/PPN?PPN=$(meta.topstruct.CatalogIDDigital)', 'http://www.kitodo.org/resolver?id=$(meta.topstruct.CatalogIDDigital)', 'http://www.kitodo.org/content/$(meta.CatalogIDDigital)/$(meta.topstruct.CatalogIDDigital).xml', 'http://www.kitodo.org/content/$(meta.firstchild.CatalogIDDigital)/$(meta.topstruct.CatalogIDDigital).xml ', 'http://www.kitodo.org/resolver?id=$(meta.CatalogIDDigital)', '', 'info@kitodo.org', '2016-01-01 00:00:00', '2019-12-31 00:00:00', 0, 0, b'0');
+/*!40000 ALTER TABLE `projekte` ENABLE KEYS */;
+
+/*!40000 ALTER TABLE `projektbenutzer` DISABLE KEYS */;
+INSERT INTO `projektbenutzer` (`BenutzerID`, `ProjekteID`) VALUES
+(1, 1),
+(2, 1),
+(3, 1),
+(4, 1),
+(5, 1),
+(6, 1);
+/*!40000 ALTER TABLE `projektbenutzer` ENABLE KEYS */;
+
+/*!40000 ALTER TABLE `projectfilegroups` DISABLE KEYS */;
+INSERT INTO `projectfilegroups` (`ProjectFileGroupID`, `name`, `path`, `mimetype`, `suffix`, `ProjekteID`, `folder`, `previewImage`) VALUES
+(1, 'MAX', 'http://www.kitodo.org/content/$(meta.CatalogIDDigital)/jpgs/max/', 'image/jpeg', 'jpg', 1, '', 0),
```

Comment:
@frank-ulrich-weber : can you make this suggested change to example.org for all needed kitodo.org entries (even mail address)?